### ISSUE TITLE
Adding flavour/personality into text

### ISSUE THREAD

**Issue by takosausage** [Treeofsavior/EnglishTranslation#569]
Hi guys, I have been reading through the current translation work since i found this repo(about 3 weeks?) and was honestly surprised at the progress made so far. Big thanks to the contributors!

Now to address the topic. As the beta is around the corner, I would like to use this opportunity to stop lurking and bring up a question/suggestion that I had in mind for a while now.
What do you, the current contributors, think about refining the text for more flavour?
For example, referring to QUEST_LV_0100.tsv:

```
QUEST_LV_0100_20150317_000004   Fi... Finally! Has help arrived?{nl}What, it's just a kid? Hey... Are there still Vubbes outside?
QUEST_LV_0100_20150317_000005   Ugh. Go away! Do not talk to me! We'll get caught by the Vubbes!{nl}This is my place so get out of here!
QUEST_LV_0100_20150317_000006   What? Do not lie to me! I can smell the Vubbes from here.{nl}You can't fool an adult! Go away! Do not come near this barrel!
```

Could be changed to

```
QUEST_LV_0100_20150317_000004   Fi... Finally! Has rescue arrived?{nl}Huh? It's just a kid? Hey... Are there still Vubbes roaming outside?
QUEST_LV_0100_20150317_000005   Damn it... Stay away! Don't talk to me! The Vubbes will notice us!{nl}This is my place so go find your own hiding spot elsewhere!
QUEST_LV_0100_20150317_000006   Seriously? Don't lie to me! I can smell the Vubbes' stench even from here.{nl}A kid like you can't fool an adult like me! Get lost! And stay away from the barrel!
```

To give off a more aloof and haughty personality. The context of the dialogue has to be further analysed in the game itself and tailored accordingly.
I understand that translation is still in its early phases and there are many things to iron out, but just wanted to throw it out here to seek opinions and insights on future work on translation.

**Comment by Nagoda** [comment]
I think we've largely avoided this kind of editing so far because the context to add more characterization is impossible to know just by looking at text files. Even just knowing what the NPC looks like is a big thing for intuiting how they might talk.

The difference between the edited examples you posted is ultimately pretty minor compared to the work that needs to be done elsewhere to help with basic legibility and grammar.

There's also the issue that if we're just arbitrarily giving all the NPCs more attitude then we're drifting towards writing fan fiction and away from translation. 

All of that said, if we do it with a SOFT TOUCH I think there is absolutely a lot of room for people to take some of the more awkwardly stilted or literal sounding language (not neccesarily "wrong") and make it sound more natural. This is ideally something that a professional localization team would have done, with the restraint to know how to not go totally overboard with it, but that's not happening.

 I think the example you made looks totally innocuous, but my main concern here is that some potential contributors will not have the writing skills or language fluency to do this sort of subtle punch up work, and the bickering that will ensue when people have differing opinions on how a character should sound. I'm not sure what the solution is when we start getting into such subjective writing territory and we have hundreds of individual contributors.

**Comment by ttgmichael** [comment]
@Nagoda's answer is spot on. Even as contributors, we still haven't played the game yet and are left out on visual context from the game, which is why it was a great thing that IMC is offering us to see the beta for ourselves. So for now, everyone has their own opinion of what a character should sound like (this is a drawback from crowdsourcing the translation, but it can be solved). Until people play the game and can draw out the context for these lines, there will be lots of disagreements on how these lines should sound (which slows down the actual translation process).

tldr; Yes, you can say the translations are still in its early-middle stages. There are still too many lines that haven't passed legibility and grammar checks (that would give you a good laugh/break the immersion playing the game), so we have to prioritize those first.  The beta will offer "real-time translation", so I guess that'll make it easier for everyone to get some of these finer details out.

**Comment by imcgames** [comment]
Yup, let's focus on legibility for now :wink:

**Comment by takosausage** [comment]
Thanks for the comprehensive feedback! Indeed I do agree with you guys, and also that the grammar and coherence of the dialogues should be done first. Just wanted to bring this topic up as something potential beta testers and contributors can look out for when the beta starts next week, since we may progress to a point where such refinement is possible before the next beta test.

**Comment by ensata** [comment]
If you've ever seen the horrors of fanfiction, you'd probably say that there are actually a lot of people who don't really know what they're doing when it comes to subtlety. If you don't know how to spot it, it's very easy to go too far and edit something to the point that it changes the fundamental meaning of the sentence. Even a single word could be important.

I can actually spot a couple of things in your example that does this with just a few words.

Saying "Ugh, go away!" can indicate that the person is disgusted and annoyed at the person in question, and doesn't really care about you.

However, saying "Damnit, stay away!" implies that the person is angry and frustrated, and may be worried about you by telling you to stay back, instead of just leaving.

Also, throwing in a pointless swear word is a shallow way of adding flavour, and should be avoided.

In the end, it's very possible that personality already exists in the original text, and adding extra stuff on top of it right now is not appropriate. It mostly comes down to the quality and accuracy of the base translation, which allows us to see it, and we're not really there yet in that area, either. I mean, yeah, I know that many games don't really have good writing, and Korean MMOs have a reputation of bad translations, but you could try giving the Korean writers the benefit of the doubt.

**Comment by takosausage** [comment]
True, at first I had the idea that a crowd-sourced project would be able to do it well, But the points raised made me realise that maybe it is because it is a crowd sourced like this I guess it would be best to leave this to actual professionals or translators who know the script well or we run the risk of veering off course(which is quite likely).

The main reason of my proposal was to see if there is a way to further increase the quality of translation as, like you said, there are many Korean MMOs out there with bad translations. :laughing: 

Now that you mentioned it though, I guess replacing "Ugh" was indeed my personal preference as it sounded unnatural to me. :stuck_out_tongue_closed_eyes:

**Comment by Ihysoal** [comment]
> This is ideally something that a professional localization team would have done, with the restraint to know how to not go totally overboard with it, but that's not happening.

Professional? Yes. Good? Probably not. I urge you to take a look at some of the dialogue from Final Fantasy XIV. They went really overboard to translate or implement their own sense of personality into the game and it's actually a really enjoyable read. To the point that a whole region has more difficult to read text because of their implied accents.

Do by all means create flavoured text and regional differences later, as it creates depth.

But that's just my 2 cents anyway. I'm hoping to be able to contribute in this way in the future at least.

**Comment by laurencee** [comment]
I don't see how having a bunch of random people on the internet (including myself) putting their own personalities into text throughout the game will result in a consistent experience.

There are so many basic translation errors right now that I don't think personality/flavour decisions should really come into the majority of pull requests at the current time.

**Comment by Ihysoal** [comment]
It's more that there's currently a lot of editors about that do not seem to have a full grasp on what proper English grammar is that will create an inconsistent experience.
